### ISSUE TITLE
2 Page roles/talents not form-fillable/editable

### ISSUE THREAD

**Issue by Felderburg** [bcholmes/StarTrek2d20#128]
Everything else in the pdf is a field that can be modified, and I think those should be too. I would suggest several large text fields, but one big one would probably work as well.

**Comment by bcholmes** [comment]
This suggestion has come up before, and I've resisted changing the behaviour. One reason has to do with the representation of the Challenge dice -- using the arrowheads wouldn't be possible in a text field because of font reasons.

**Comment by Felderburg** [comment]
Right, fair enough. It would just show up as A if you don't have the font from the official books... but if pdfs support emojis, you could always use the 🖖emoji (vulcan salute emoji if it doesn't show up here). Although there might be a way to use multiple fonts in a single text box: https://community.adobe.com/t5/acrobat-discussions/use-multiple-font-options-within-one-field/m-p/9678655#M89600

**Comment by TakeshiYamato01** [comment]
You could always replace the arrowheads with 'Challenge Dice', that being what they represent. Or even the shorthand form of 'CD'.
